### ISSUE TITLE
versioning plugins

### ISSUE THREAD

**Issue by danielepolencic** [dokku/dokku#2232]
My server is provisioned by a bash script that installs dokku (0.4.1) and few plugins (redis, mariadb).

I wrote the script 9 months ago and at the time there was no way to version plugins.

Fast forward to today, when I run the script locally I have few errors to do with the mariadb plugin:

```
/var/lib/dokku/plugins/enabled/mariadb/functions: line 5: /var/lib/dokku/plugins/available/docker-options/functions: No such file or directory
```

The same plugin works fine in the sever provisioned 9 months ago.

I very much appreciate that dokku and dokku-mariadb evolved in this 9 months period. But is there any way for me to specify a plugin version (or commit)?

I had a look at dokku/plugn but I realised that the install command is a simple `git clone`. I'm not sure that tagging/versioning is supported.

Any idea on how I could checkout an older version of the plugin?

**Comment by josegonzalez** [comment]
You can do so as specified [here](http://dokku.viewdocs.io/dokku/plugins/#installing-a-plugin) using the `--committish` flag:

```shell
sudo dokku plugin:install https://github.com/dokku/dokku-mariadb.git --committish SOME_COMMIT
```

You may also wish to file that specific issue as a bug in the plugin, since we ostensibly still support 0.4.x in the official plugins.

**Comment by danielepolencic** [comment]
As a reference, `--committish` is available from 0.4.10

**Comment by josegonzalez** [comment]
Note: I've opened PRs for the relevant issue in the plugin repos and will merge later today.
